api/envoy/config/trace/v2/trace.proto

```diff
@@ -22,11 +22,22 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#protodoc-title: Tracing]
 // Tracing :ref:`architecture overview <arch_overview_tracing>`.
 
-// The tracing configuration specifies global
-// settings for the HTTP tracer used by Envoy. The configuration is defined by
-// the :ref:`Bootstrap <envoy_api_msg_config.bootstrap.v2.Bootstrap>` :ref:`tracing
-// <envoy_api_field_config.bootstrap.v2.Bootstrap.tracing>` field. Envoy may support other tracers
-// in the future, but right now the HTTP tracer is the only one supported.
+// The tracing configuration specifies settings for an HTTP tracer provider used by Envoy.
+//
+// The configuration should preferably be defined as part of
+// the :ref:`HttpConnectionManager.Tracing <envoy_api_msg_config.filter.network.http_connection_manager.v2.HttpConnectionManager.Tracing>`
+// :ref:`provider <envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.Tracing.provider>`
+// field.
+//
+// Alternatively, for the sake of short-term backwards compatibility, it is also possible to define
```

Comment:
Same comment here. Please make the new docs refer to the expected new config, and talk about deprecation in deprecated.rst

Reply:
fixed